sirepo/template/radia_util.py

```diff
@@ -303,6 +303,14 @@ def build_cuboid(**kwargs):
     return g_id
 
 
+def build_stl(**kwargs):
+    d = PKDict(kwargs)
+    f = numpy.array(d.faces)+1
```

Comment:
Spaces around operators. Even better, adjust the faces array before you pass it. Also, no need for a temp variable:
```python
g_id = radia.ObjPolyhdr(
    d.vertices,
    (numpy.array(d.faces) + 1).tolist(),
    d.magnetization
)
```